pom.xml

```diff
@@ -6,7 +6,7 @@
   <artifactId>amazon-kinesis-client</artifactId>
   <packaging>jar</packaging>
   <name>Amazon Kinesis Client Library for Java</name>
-  <version>1.14.4-SNAPSHOT</version>
+  <version>1.14.5</version>
```

Comment:
Change log says version 1.14.4?

Reply:
Changed in [this](https://github.com/awslabs/amazon-kinesis-client/pull/821/commits/27b5df15315c15d9a6bb090abb1d39f6344df8c9) commit